examples/tree_2d_dgsem/elixir_mhdmultiion_ec.jl

```diff
@@ -0,0 +1,60 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal MHD equations
+equations = IdealMhdMultiIonEquations2D(gammas           = (2.0, 2.0),
+                                                 charge_to_mass   = (1.0, 1.0))
```

Comment:
```suggestion
equations = IdealMhdMultiIonEquations2D(gammas = (2.0, 2.0),
                                        charge_to_mass = (1.0, 1.0))
```
Same change for the other elixirs